jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/WebAppContext.java

```diff
@@ -1438,13 +1439,11 @@ public URL getResource(String path) throws MalformedURLException
                 return null;
 
             // Should we go to the original war?
-            if (resource.isDirectory() && resource instanceof ResourceCollection && !WebAppContext.this.isExtractWAR())
+            if (resource.isDirectory() && !WebAppContext.this.isExtractWAR())
```

Comment:
Same comment as for ee10, review this logic.

Reply:
Done